plugins/obs-stream-effects/source/version.h.in

```diff
@@ -0,0 +1,25 @@
+/*
+ * A Plugin that integrates the AMD AMF encoder into OBS Studio
+ * Copyright (C) 2016 - 2017 Michael Fabian Dirks
```

Comment:
Copyright years are generally meant to cover years of public releases of software, so this should probably be just 2017 like the rest of the files.